Otto/readme.md

```diff
@@ -0,0 +1,10 @@
+# Otto
+
+## Screenshots
+
+![Otto1](https://raw.githubusercontent.com/ignaciocastro/spicetify-themes/master/Otto/Otto.png)
+![Otto2](https://raw.githubusercontent.com/ignaciocastro/spicetify-themes/master/Otto/Otto2.png)
```

Comment:
Please use relative paths (look at other readmes which refer to pictures in the same folder)